PR for material folder and input folder moved to Skvt folder

Aeinahpets opened the pull request.

Aeinahpets committed:
Move materials

move from /Content/Characters/Mannequins/Materials/ to  /All/Game/Skvt/Characters/Materials/
Yellow and green material instance created

Aeinahpets committed:
Move default input mapping to skvt folder

moved /Content/ThirdPerson/Input/ to /Game/Skvt/Input

RavenGoddess reviewed (changes requested):
Green and yellow instances do not follow the standards set by other material instances:
naming convention MI_Manny_01 and MI_Manny_02_color

Furthermore it appears there is only one of the two materials required for the full character

MI_Quinn materials do not have any material color instances MI_Manny has, please add them.

The characters, such as NPC_base and Skvt_base should be resaved, ideally with redirectors cleaned up

Aeinahpets committed:
green yellow naming conventions, input mapping

Change yellow and green naming conventions.
NPC_base and Skvt_base input mapping redirected.

Aeinahpets committed:
Redirector cleanup

redirector cleanedup, materials missing applied to thirdperson character and npcs
